### ISSUE TITLE
Does not get enabled after install in 3.38

### ISSUE THREAD

**Issue by WuerfelDev** [brunelli/gnome-shell-extension-installer#16]
I don't know if it's just me, but running a command looks like this:

```
$ gnome-shell-extension-installer 3518
[3518] Obtaining extension info
[Transparent Shell] Downloading extension
[Transparent Shell] Extracting extension
Extension “transparent-shell@siroj42.github.io” does not exist
```

The last line comes from `gnome-extensions enable transparent-shell@siroj42.github.io`. I noticed that the shell needs to be restarted before running the command.

Maybe it is possible to add the extension manually to `settings get org.gnome.shell enabled-extensions` ?

**Comment by brunelli** [comment]
> Maybe it is possible to add the extension to `settings get org.gnome.shell enabled-extensions` instead of the `gnome-extensions` command ?

Yeah, [that's what I was doing before](https://github.com/brunelli/gnome-shell-extension-installer/commit/aa6d2d283a66ce09c47fa5a5071b45f6661a5cb3).

Unfortunately, I'm not running GNOME Shell anymore, so this project is probably going unmaintained soon.

**Comment by vonschutter** [comment]
Just add chmod -R 755 /usr/share/gnome-shell/extensions and restart gnome-shell... This fixes the issue for global installs.
